Entity/OverrideLeadFieldRepository.php

```diff
@@ -234,7 +235,16 @@ private function getExtendedField($alias)
             )
             ->setParameter('alias', $alias);
 
-        return $qf->execute()->fetch();
+        $stmt = $qf->getConnection()->executeCacheQuery(
+            $qf->getSQL(),
+            $qf->getParameters(),
+            $qf->getParameterTypes(),
+            new QueryCacheProfile(60, 'get-extended-field')
```

Comment:
The alias needs to be part of the cache key or this will not work when getting a second extended field. Right?

If we were running a query to get ALL and caching that array, that'd be different... That's not what is happening here.

Reply:
changed